### DISCUSSION TITLE
Welcome to our 2023 Google Season of Docs tech writer!

### DISCUSSION THREAD

**Discussion by kathatherine** [conda/conda-dot-org#125] [Announcements]
I am happy to announce that after weeks of searching and filtering through applications, the conda.org team has chosen a tech writer for our 2023 Google Season of Docs proposal.

Please welcome [Pinak Datta](https://github.com/Pinak-Datta)!

![pinak_photo](https://user-images.githubusercontent.com/90850002/236914358-bb2c902a-f5d5-4b44-bb2b-c1d74fd9e4f6.png)

Pinak is a tech writer from India who is currently in his third year at the Kalinga Institute of Industrial Technology. His first experience with conda was through the Anaconda repository, but he stayed because of the community. We are excited to see how his experience with the tool and his experience as a student will help us make the conda organization an accessible place for users of all kinds, from the newbie to the experienced.

From Pinak: "Hi, I'm Pinak, a pre-final year undergraduate student studying Computer Science and Engineering. I have a strong passion for technical writing and believe that clear and concise documentation is essential for helping users understand a tool's functionality.

In my free time, I love creating technical blogs and tutorials to help others learn new skills and technologies. I'm excited to be joining the [conda.org](http://conda.org/) team as a tech writer for the 2023 Google Season of Docs, and I'm looking forward to working with the community to make conda more accessible to users of all experience levels."

**Comment by asmitbm** [comment]
Welcome @Pinak-Datta! 🥳

Glad to have you on the team. Looking forward to work with you :)
